testbed/testbed/senders.go

```diff
@@ -593,7 +593,7 @@ func (pds *PrometheusDataSender) Start() error {
 		return err
 	}
 
-	pds.MetricsConsumer = exp
+	pds.Metrics = exp
```

Comment:
This is worse than before. It is not clear what this `Metrics` is since it is a very generic word. I do not mind renaming the type names but if we do that we must ensure embedded structs become fields so that they have a proper name otherwise readability of the code suffers.

Reply:
I do agree that embedded fields are not best named in this situation, I would still do the change because it is more go standard way to name things like this. Do you mind if I look into this in a followup?